readme.md

```diff
@@ -81,7 +82,10 @@ Your Rails database has two tables. `students` has the columns `id` and `name`,
 Use ActiveRecord to create a new `pbj` sandwich and make it belong to the student named Geraldo.
 
 ```rb
-# Your answer...
+student = student.new(name: "Geraldo", id: 1);
```

Comment:
Remember we don't specify IDs when creating things, as the DB will do so for us.

```ruby
geraldo = Student.create(name: "Geraldo")
sammich = Sandwich.create(type: "obj", student: geraldo)
# or 
sammich = Sandwich.create(type: "obj", student_id: geraldo.id)
```